locales/sq_AL/LC_MESSAGES/duckduckgo.po

```diff
@@ -474,69 +476,69 @@ msgstr "Bazë"
 
 msgctxt "theme"
 msgid "Basic"
-msgstr ""
+msgstr "Tema"
 
 msgctxt "precise_user_location"
 msgid "Before you go, please help us understand why:"
-msgstr ""
+msgstr "Para se të na lini, ju lutem na ndihmoni të kuptojmë pse"
 
 msgid "Belgium (fr)"
-msgstr ""
+msgstr "Belgjikë"
 
 msgid "Belgium (nl)"
-msgstr ""
+msgstr "Belgjikë"
 
 msgctxt "settingsvalue"
 msgid "Best Available"
-msgstr ""
+msgstr "Versioni më i mirë"
 
 #. In the right menu panel of the homepage.
 msgid "Beta"
-msgstr ""
+msgstr "Beta"
 
 msgctxt "video-duration"
 msgid "Between 4 and 20 minutes"
-msgstr ""
+msgstr "Mes 4 dhe 20 minutave"
 
 #. The color black.
 msgid "Black"
 msgstr "Zezë"
 
 msgctxt "color"
 msgid "Black"
-msgstr ""
+msgstr "Zezë"
 
 msgctxt "image-color"
 msgid "Black and White"
-msgstr ""
+msgstr "Bardh e Zi"
 
 msgid "Block Trackers"
-msgstr ""
+msgstr "Blloko përgjuesit"
 
 msgctxt "reasons-to-use-duckduckgo"
 msgid "Block advertising trackers."
-msgstr ""
+msgstr "Blloko reklamat me përgjime"
```

Comment:
Please, keep an eye out for punctuation. In this case, the string should end with a `.` period. If you use specialised software for translations, such as [Poedit](https://poedit.net), it will highlight such formatting inconsistencies for you. It's not a requirement, of course, just a suggestion 😉. I use that one, but there are others. Thanks!

```suggestion
msgstr "Blloko reklamat me përgjime."
```